lib/faraday/connection.rb

```diff
@@ -423,7 +423,7 @@ def build_url(url = nil, extra_params = nil)
     #
     # @param method [Symbol] HTTP method.
     # @param url [String, URI, nil] String or URI to access.
-    # @param body [String, nil] The request body that will eventually be converted to
+    # @param body [Object, nil] The request body that will eventually be converted to
     #             a string.
```

Comment:
```suggestion
    # @param body [String, Hash, Array, nil] The request body that will eventually be converted to
    #             a string; middlewares can be used to support more complex types.
```

Reply:
@iMacTia yup that makes sense, how's this? ^^

Reply:
looks great 🙌 !